pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1077,5 +1078,76 @@ private void TestRedundantComboBox()
             }
             RunUI(() => dlg.Close());
         }
+
+        // Test for property sheet

```

Comment:
Make this a block comment with useful text. This just repeats the function name adding nothing useful.